cmd/opdoc/opdoc.go

```diff
@@ -28,13 +28,13 @@ import (
 	"github.com/algorand/go-algorand/protocol"
 )
 
-var docVersion = 8
+var latestVersion = 8
```

Comment:
If we're going to generate all versions, including the one after the current consensus version (which I think is correct). We can go back to a world that does not have this variable at all, and uses the value in `logic.LogicVersion`.

If we generate all these langspecs, we'll have to change the website to pick up the right one to show. (Maybe it should show them all somehow, but the "main" one should be the current mainnet's consensus value.